docs/getting-started.md

```diff
@@ -22,26 +22,64 @@ yarn add offix-client
 
 ```javascript
 import { OfflineClient } from 'offix-client';
-```
 
-## Configuration
+const config = {
+  httpUrl: 'http://localhost:4000/graphql'
+};
 
-To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `OffixClientConfig` interface from the SDK.
+// offlineClient is a parent around the apollo client
```

Comment:
What this means?

Reply:
Sorry maybe bad language. `OfflineClient` class really is a container/wrapper that has the extended apolloClient as a property

Reply:
Not really sure how to get that across

Reply:
Ah.. right. Sorry it is hard to actually understand this sentence. Maybe 2 sentences?